### ISSUE TITLE
[Feature]: Pop-up button with additional features

### ISSUE THREAD

**Issue by iG8R** [pot-app/pot-desktop#465]
### Please confirm the following before submitting a new issue.

- [X] I have carefully reviewed the official website's usage documentation.
- [X] I have searched through the historical issues but could not find an answer.

### Description

Could you, please, implement a pop-up button after selecting a text, as shown in the attached video?
Also, as you could see there appear a button panel after hovering over pop-upped button with additional features such as (from left to right): open dictionary window, read the selected text aloud, copy the selected text to clipboard, replace the selected text with the translated one.

[Pot-Translator-PopUpButton-Video_2023-09-14_174048FFB.webm](https://github.com/pot-app/pot-desktop/assets/11407417/3ccea68d-e34a-4551-bc56-62d6aa6daad1)

### Application Scenario

For convenience use.

**Comment by Pylogmon** [comment]
https://github.com/pot-app/pot-desktop#%E7%8E%B0%E6%9C%89%E7%94%A8%E6%B3%95-%E5%BF%AB%E6%8D%B7%E5%88%92%E8%AF%8D%E7%BF%BB%E8%AF%91

**Comment by iG8R** [comment]
Yeah, I saw this in documentation, but... long time ago I bought life-time license for SnipDo but the author cancels all such licenses
and  switched to a new system of earning - subscription, dumping those who once supported him, so there is no desire to use this app further because it reminds of such an act of the author.
